weather.py

```diff
@@ -395,10 +297,10 @@ def display_subwindow(self, data, day, c_times):
                                             line_spacing_gap * 0)))
         if hasattr(data, 'temperatureLow'):
             txt = forecast_font.render(
-                str(int(round(data.temperatureLow))) +
+                str(int(round(data.temperatureHigh))) +
                 UNICODE_DEGREE +
                 ' / ' +
-                str(int(round(data.temperatureHigh))) +
+                str(int(round(data.temperatureLow))) +
```

Comment:
I don't disagree with this, but I find it interesting that after we chatted Darksky updated their app and website to show the low first. Do you think we should revisit changing this? (honest question)

Reply:
So, I went to two different weather websites and looked at highs and lows for the day. On Darksky (and the others) the high corresponded to the mid-afternoon on the day and the low corresponded to the early morning the next day. I think it's very counter-intuitive, but it seems like all weather forecasters adhere to this convention. In case I've misread something, though. Would you check by opening a couple different weather sites and confirm that the low for the day is, in fact, the temperature for the next days' early morning. You'll have to check the hour-by-hour.

Reply:
We could also consider making this a configurable option.

Reply:
Let's just make it configurable